static/app/components/replays/player/scrubber.tsx

```diff
@@ -63,11 +65,21 @@ function Scrubber({className, showZoomIndicators = false}: Props) {
         {currentHoverTime ? (
           <div>
             <TimelineTooltip
-              labelText={formatDuration({
-                duration: [currentHoverTime, 'ms'],
-                precision: 'ms',
-                style: 'hh:mm:ss.sss',
-              })}
+              labelText={
+                timestampType === 'absolute'
+                  ? getFormattedDate(
+                      startTimestamp + currentHoverTime,
+                      shouldUse24Hours() ? 'HH:mm:ss.SSS' : 'hh:mm:ss.SSS',
```

Comment:
I don't think we need the AM/PM in the scrubber since the timeline has AM/PM, but if you think it'll be better to have in the scrubber too, we can add it in as well!

Reply:
Oo didn't notice that, in that case I'm fine either way!